tests/framework/extensions/psact/createdeployment.go

```diff
@@ -21,6 +21,7 @@ const (
 	namespace         = "default"
 	rancherPrivileged = "rancher-privileged"
 	rancherRestricted = "rancher-restricted"
+	rancherBaseline   = "rancher-baseline"
```

Comment:
this Const in a few places. Can we remove it from here and import from provisioninginput?

Reply:
Agreed